### ISSUE TITLE
improved discarding of message from mapping script

### ISSUE THREAD

**Issue by GoogleCodeExporter** [dmanty45/bots#10]
```
it is possible to discard a message form the mapping script.
but it is needed to write 'dummy data'; and this data has to be valid (for the 
outgoing message format).
thi is imporved: it is no longer needed to write the dummy data (btw, it is 
allowed to do this).

```

Original issue reported on code.google.com by `hjebb...@gmail.com` on 13 Jul 2010 at 1:22
- Blocked on: #20100712

**Comment by GoogleCodeExporter** [comment]
Original comment by `hjebb...@gmail.com` on 13 Jul 2010 at 1:29
- Added labels: **Type-Improve**
- Removed labels: **Type-Bug**

**Comment by GoogleCodeExporter** [comment]
Original comment by `hjebb...@gmail.com` on 14 Jul 2010 at 6:01
- Added labels: **Version2.0.1**
- Removed labels: **Priority-Medium**

**Comment by GoogleCodeExporter** [comment]
Original comment by `hjebb...@gmail.com` on 26 Jul 2010 at 9:34
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
Original comment by `hjebb...@gmail.com` on 10 Sep 2013 at 12:44
- Changed state: **Done**
- Now blocked on: #20100712
